### PR TITLE
docs(setup): Clarifies the need to invoke hal deploy apply in order t…

### PR DESCRIPTION
**Pull request by dorbin**
…o enable Igor

### CODE REVIEW

**Comment by dorbin** [comment]
@jtk54 I included you cause I made a small change to your CSRF section.

**Review by lwander — APPROVED** [review]

**Comment by dorbin** [comment]
#922 
@tillig

File: setup/ci/jenkins.md [hunk 1]
```diff
@@ -121,29 +129,37 @@ In the UI, go to `"Manage Jenkins"` >> `"Configure System"` and set your git `us
...
-```
-jenkins.enabled = true
-jenkins.masters[0].name = <jenkins_name>
-jenkins.masters[0].address = http://<jenkins_host>/jenkins Note that "/jenkins" might not be the base path, it depends on how Jenkins is configured.
-jenkins.masters[0].username = <username>
-jenkins.masters[0].password = <user_api_token>
-```
+   Because you [enabled Jenkins](#add-your-jenkins-master), Igor should be
+   enabled automatically by Spinnaker. You can confirm this by checking your
+   `~/.hal/default/service-settings/`
```
Comment (tillig) [inline setup/ci/jenkins.md:138]: What should I see in here? Should it have created an `igor.yml`? (I still don't have one, though it appears Igor is enabled.)

Comment (dorbin) [reply setup/ci/jenkins.md:138]: I think you would only create an igor.yml if you needed to explicitly override an igor enabled:false.

Sorry, I merged *just* before I saw this question.

File: setup/ci/jenkins.md [hunk 2]
```diff
@@ -121,29 +129,37 @@ In the UI, go to `"Manage Jenkins"` >> `"Configure System"` and set your git `us
...
+   `~/.hal/default/service-settings/igor.yml`:
 
-```
-script:
-  master: <jenkins_name> # name of Jenkins master in Spinnaker
-  job: <JOB_NAME> # from Jenkins job configuration
-```
+   `enabled:true`
+
+1. In spinnaker-local.yml, set:
```
Comment (tillig) [inline setup/ci/jenkins.md:145]: Is this needed _in addition to_ the `hal config ci jenkins master add` call that was made earlier? (Same question for the `orca-local.yml` below.)

Comment (dorbin) [reply setup/ci/jenkins.md:145]: Yeah, that's a good point. That might be vestigial, from before halyard. I'll have a new PR ready in the morning. 
Thanks!

Comment (dorbin) [reply setup/ci/jenkins.md:145]: @lwander

**Pull request merged**